src/main/config/messages_template.xml

```diff
@@ -7,413 +7,415 @@ Copyright 2005, 2006 IBM Corporation
+    <response>For correct processing, make sure the input is normalized DITA with @class attributes included.</response>
   </message>
-  
+
   <message id="DOTJ038E" type="ERROR">
-    <reason>The tag "%1" is specialized from unrecognized metadata.</reason>
-    <response>Please make sure that tag "%1" is specialized from an existing metadata tag in the core DITA vocabulary.</response>
+    <reason>The &lt;%1&gt; element is specialized from unrecognized metadata.</reason>
+    <response>Make sure that the &lt;%1&gt; element is specialized from an existing metadata element in the core DITA vocabulary.</response>
   </message>
-  
+
   <message id="DOTJ039E" type="ERROR">
-    <reason>There is no target specified for conref push action "pushafter".</reason>
-    <response>Please add &lt;elementname conref="pushtarget" conaction="mark"&gt; before current element.</response>
+    <reason>There is no target specified for the conref push action 'pushafter'.</reason>
+    <response>Specify a @conref target and set the 'mark' @conaction on the element that precedes the current element.</response>
   </message>
-  
+
   <message id="DOTJ040E" type="ERROR">
-    <reason>An element uses the attribute conaction="replace", but a conref attribute is not found in the expected location.</reason>
-    <response></response>
+    <reason>An element uses the conref push action 'pushreplace', but no @conref attribute is defined.</reason>
+    <response>Specify a @conref target with the ID of the content you want to replace.</response>
   </message>
-  
+
   <!-- Note, the following message comes from the Java conref-push module, XSL based message later catches regular conref (DOTX014E and DOTX015)-->
   <message id="DOTJ041E" type="ERROR">
-    <reason>The attribute conref="%1" uses invalid syntax.</reason>
-    <response>The value should contain '#' followed by a topic or map ID, optionally followed by '/elemID' for a sub-topic element.</response>
+    <reason>The @conref attribute value '%1' uses invalid syntax.</reason>
+    <response>The value must contain '#' followed by a topic or map ID, optionally followed by '/elemID' for a sub-topic element.</response>
   </message>
-  
+
   <message id="DOTJ042E" type="ERROR">
-    <reason>Two elements both use conref push to replace the target "%1".</reason>
-    <response>Please delete one of the duplicate "replace" actions.</response>
+    <reason>Two elements both use conref push to replace the target '%1'.</reason>
+    <response>Delete one of the duplicate 'pushreplace' actions.</response>
   </message>
-  
+
   <message id="DOTJ043W" type="WARN">
-    <reason>The conref push function is trying to replace an element that does not exist (element "%1" in file "%2").</reason>
-    <response></response>
+    <reason>The conref push function is trying to replace an element &lt;%1&gt; that does not exist in the '%2' resource.</reason>
+    <response>Update the reference to point to a valid target.</response>
```

Comment:
Nit ... should we use "to reference" rather than "to point to"?

Reply:
Was trying to avoid “Update the reference to reference…”